### PR TITLE
Add getting started with queue as a service blogpost

### PR DESCRIPTION
**Pull request by guillaumebreton**
The first blog post for PaaS queue :) feel free to comment @NicolasLM @yadutaf

### CODE REVIEW

**Comment by NicolasLM** [comment]
I would add a simple schema of producer/consumer, just so people not familiar with this pattern understand what it is about.

File: en/_posts/2016-06-15-getting-started-with-queue-as-a-service.md [hunk 1]
```diff
@@ -0,0 +1,123 @@
+---
+layout: post
+title:  "Getting started with OVH PaaS Queue"
```
Comment (yadutaf) [inline en/_posts/2016-06-15-getting-started-with-queue-as-a-service.md]: I think the correct English is "Queue PaaS" (Word swap)

File: en/_posts/2016-06-15-getting-started-with-queue-as-a-service.md [hunk 2]
```diff
@@ -0,0 +1,123 @@
...
+categories: queue
+author: guillaumebreton
+lang: en
+---
+# OVH PaaS Queue and Kafka 101
+
+Welcome to the getting started of the OVH PaaS Queue. This guide will help you to understand the core concepts behind the OVH PaaS Queue
```
Comment (yadutaf) [inline en/_posts/2016-06-15-getting-started-with-queue-as-a-service.md]: "To the getting started guide"

File: en/_posts/2016-06-15-getting-started-with-queue-as-a-service.md [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+and how to produce your first message to the platform and consume it.
+
+## What is the OVH PaaS Queue ?
+
+The OVH PaaS Queue provides you a mutualized queuing system based on [Apache Kafka](http://kafka.apache.org/).
```
Comment (yadutaf) [inline en/_posts/2016-06-15-getting-started-with-queue-as-a-service.md]: I wonder If "shared" would not be a better word for "mutualized" but I ain't native either

File: en/_posts/2016-06-15-getting-started-with-queue-as-a-service.md [hunk 4]
```diff
@@ -0,0 +1,123 @@
...
+
+It powers fast, scalable and global low-latency messaging to allow you to connect anything to everything.
```
Comment (yadutaf) [inline en/_posts/2016-06-15-getting-started-with-queue-as-a-service.md]: - It provides
- allowing you to

File: en/_posts/2016-06-15-getting-started-with-queue-as-a-service.md [hunk 5]
```diff
@@ -0,0 +1,123 @@
...
+It allows you to connect producers and consumers through topics with fault tolerance as well as journalized events.
+All messages are persisted on disk and replicated within the cluster to prevent data loss.
+
+## What is a producer and a consumer ?
+
+Kafka maintains feeds of messages in topics. Producers write data to topics and
+consumers read from topics. Since Kafka is a distributed system, topics are partitioned
+and replicated across multiple nodes.
+Each message in a partition is assigned a unique offset. A consumer can save its offset position
+to remember the messages read.
```
Comment (yadutaf) [inline en/_posts/2016-06-15-getting-started-with-queue-as-a-service.md]: Can you detail the persistence model ? This is a bit different from what most users are used to when using RabbitMQ or Node.JS' Socket.IO where messages are destroyed after being consumed. Maybe a work on the distributed log paradigm would help ?

File: en/_posts/2016-06-15-getting-started-with-queue-as-a-service.md [hunk 6]
```diff
@@ -0,0 +1,123 @@
...
+
+# Getting started
+
+The OVH PaaS Queue has been built so that you can connect painlessly to it.
+Topics are automatically created at the first request and a partitionning strategy is applied by default.
```
Comment (yadutaf) [inline en/_posts/2016-06-15-getting-started-with-queue-as-a-service.md]: "a partitioning strategy" --> typo + could you detail which strategy is deployed ? Tech details helps to get confidence in the tech

File: en/_posts/2016-06-15-getting-started-with-queue-as-a-service.md [hunk 7]
```diff
@@ -0,0 +1,123 @@
...
+    ```
+
+3. Start a producer and write something to stdin to produce one message (the topic is automatically created).
+
+    ```
+    ./qaas-client-$(uname -s)-amd64 produce --kafka $HOST:9092 --key $KEY --topic $PREFIX.$TOPIC
+    # Then write to STDIN to send a message
+    ```
+
+# Languages supported
```
Comment (yadutaf) [inline en/_posts/2016-06-15-getting-started-with-queue-as-a-service.md]: "Supported languages" (word swap)

File: en/_posts/2016-06-15-getting-started-with-queue-as-a-service.md (hunk 5 above)
Comment (guillaumebreton) [inline en/_posts/2016-06-15-getting-started-with-queue-as-a-service.md]: It's a good idea, but we will do it in an other episode :)

**Comment by guillaumebreton** [comment]
Thx for all this feedback 👍 I updated the page

**Pull request merged**